src/components/postComments/postComments.js

```diff
@@ -0,0 +1,30 @@
+import { Fragment, useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { getPostComments } from "../../services/api";
+
+const PostComments = () => {
+  const [comments, setComments] = useState([]);
+  const { postId } = useParams();
+  const loadPostComments = async () => {
+    try {
+      const response = await getPostComments(postId);
+      const { data } = response;
+      setComments(data);
+    } catch (error) {
+      throw new Error("Could't get post's comments");
+    }
+  };
+
+  useEffect(() => {
+    loadPostComments();
+  }, [postId]);
```

Comment:
use some validations to validate postId

Reply:
parseInt(postId), is this the right approach?